test-evidence/BUILD.bazel

```diff
@@ -101,5 +100,6 @@ genrule(
         cd $$DIR
         $$MKTGZ $$OUT daml-test-evidence
         """,
-    tools = ["//bazel_tools/sh:mktgz"],
-)
+        tools = ["//bazel_tools/sh:mktgz"],
+    ),
+] if is_linux else []
```

Comment:
@skisel @mziolekda Are you fine with that ?

Reply:
I think it should be okay, as release is done in unix